crates/libeq_wld/src/parser/mod.rs

```diff
@@ -49,6 +50,30 @@ impl WldDoc {
         ))
     }
 
+    pub fn fragment_headers_by_offset(input: &[u8]) -> BTreeMap<usize, FragmentHeader> {
```

Comment:
This has been sort of useful for debugging some issues. Not sure it'll hang around in this form but here it is for now. `wld-cli stats` uses this.